community/cloud-foundation/templates/iam_member/README.md

```diff
@@ -1,11 +1,12 @@
 # IAM Member
 
-IAM Member deployment
+This template creates an IAM member.

```

Comment:
This templates adds the members only at project level right? We should clarify that as well.

Reply:
@alexander-levitski This template creates grants IAM roles for a project.